### PR TITLE
Add Timed Unit Toggling & Minor Fix to Error Message

### PR DESCRIPTION
**Pull request by rogadev**
Small improvement to the error message including styling to ensure you can see the message on screen. Plus added the MIT license to the package.json.

### CODE REVIEW

**Review by drillskibo — CHANGES_REQUESTED** [review]
You should not *remove* the package-lock but just not update it on the PR. You should keep the current version on main branch.

**Comment by rogadev** [comment]
> You should not _remove_ the package-lock but just not update it on the PR. You should keep the current version on main branch.

Gotchya. What is the easiest way to keep the old version of the package-lock?

**Review by drillskibo — COMMENTED** [review]
You might want to check [#1](https://github.com/rogadev/weather-overlay/pull/1).

**Review by drillskibo — APPROVED** [review]
LGTM ! 👍🏼

File: src/App.svelte [hunk 1]
```diff
@@ -48,10 +55,15 @@
           src={`images/svg/${currentWeather.symbol_code}.svg`}
         />
       </div>
-      <div class="temperature">
-        {currentWeather.temperature.F}<span class="unit">°F</span>
-        <!-- {currentWeather.temperature.C} <span class="unit">°C</span> -->
-      </div>
+      {#if !showMetric}
+        <div class="temperature">
+          {currentWeather.temperature.F} <span class="unit">°F</span>
```
Comment (w3cj) [inline src/App.svelte]: Please remove space before span.

File: src/App.svelte [hunk 2]
```diff
@@ -48,10 +55,15 @@
...
+        </div>
+      {:else}
+        <div in:fade class="temperature">
+          {currentWeather.temperature.C} <span class="unit">°C</span>
```
Comment (w3cj) [inline src/App.svelte]: Please remove space before span.

**Review by w3cj — CHANGES_REQUESTED** [review]

File: src/App.svelte [hunk 3]
```diff
@@ -1,16 +1,21 @@
...
+  import { fly, fade } from 'svelte/transition';
   import type { WeatherResult } from "./lib/weatherAPI";
   import { getWeather } from "./lib/weatherAPI";
 
   let currentWeather: WeatherResult | null = null;
   let weatherVisible = false;
   const params = new URLSearchParams(window.location.search);
   let errorMessage = "";
+  let showMetric = false;
+  let showDuration = 20000;
```
Comment (w3cj) [inline src/App.svelte]: Please default to 10 * 1000 instead of 20 * 1000

**Review by drillskibo — APPROVED** [review]
Fixed requested changes.

File: src/App.svelte (hunk 1 above)
Comment (drillskibo) [reply src/App.svelte]: ✅

File: src/App.svelte (hunk 2 above)
Comment (drillskibo) [reply src/App.svelte]: ✅

File: src/App.svelte (hunk 3 above)
Comment (drillskibo) [reply src/App.svelte]: ✅

**Comment by w3cj** [comment]
I'd prefer if the text animated out and in.

```svelte
<div class="temperature">
  {#if !showMetric}
    <span out:fade={{ duration: 800 }}>
      {currentWeather.temperature.F}<span class="unit">°F</span>
    </span>
  {:else}
    <span in:fade={{ duration: 800 }}>
      {currentWeather.temperature.C}<span class="unit">°C</span>
    </span>
  {/if}
</div>
```

```css
  .temperature {
    display: grid;
  }

  .temperature span {
    grid-area: 1/1;
  }
```

**Review by w3cj — APPROVED** [review]

**Pull request merged**